java-hello-world/README-Local-macOS.md

```diff
@@ -0,0 +1,196 @@
+# Java Hello World with GraalVM Enterprise on macOS
+
+This sample shows how you can get started quickly with GraalVM Enterprise Edition on a local machine running macOS. 
```

Comment:
Why do we need "Java Hello World with GraalVM Enterprise on macOS"? The difference between this one and https://www.graalvm.org/22.2/docs/getting-started/#run-java is that the project uses a Maven Plugin. 
NI Maven and Gradle plugins are used in other demos, so tested for compatibility. 
As a variant, we can create a project native-image-build-tools-demo and use a simple Java app built with Maven and Gradle.

Reply:
Removed local macOS.